cmd/webhook/main.go

```diff
@@ -98,7 +103,7 @@ func NewValidationAdmissionController(ctx context.Context, cmw configmap.Watcher
 
 		// A function that infuses the context passed to Validate/SetDefaults with custom metadata.
 		func(ctx context.Context) context.Context {
-			return ctx
+			return contexts.WithUpgradeViaDefaulting(store.ToContext(ctx))
```

Comment:
`contexts.WithUpgradeViaDefaulting`

Are you using this upgrade marker for anything?

Reply:
yeah we do